helper/codec/inmem.go

```diff
@@ -20,6 +20,9 @@ func (i *InmemCodec) ReadRequestHeader(req *rpc.Request) error {
 }
 
 func (i *InmemCodec) ReadRequestBody(args interface{}) error {
+	if args == nil {
+		return nil
+	}
```

Comment:
Interesting... is this an existing panic bug? Or is it the kind of thing we can only ever hit in testing?

Reply:
If code makes it out if the shop with an unwired RPC, it would panic.  It's unlikely because of testing, but the panic hides the error from being output when the test fails; so I lost a lot of time step-debugging what would have been a comparatively simple error fix.

Reply:
Good catch then, that makes life easier for everyone going forward :+1: